core/src/test/java/org/axonframework/eventhandling/TrackingEventProcessorTest.java

```diff
@@ -711,38 +711,6 @@ public void testHasAvailableSegments() {
         assertWithin(2, TimeUnit.SECONDS, () -> assertEquals(1, testSubject.availableProcessorThreads()));
     }
 
-    @Test
```

Comment:
This fix you've provided earlier (thanks again for that) should be merged into `3.4.x`. So updating your branch should remove this change.